CHANGELOG.rst

```diff
@@ -2,6 +2,30 @@
 Changelog
 =========
 
+0.28.0 (unreleased)
+===================
+
+* Added official Python 3.14 support.
+
+* Switched attribute documentation from Sphinx comments (``#:``) to docstrings.
+
+  This allows IDEs to show them when hovering attributes, and allows
+  `ItemAdapter.get_json_schema()`_ to include them as descriptions.
+
+  .. _ItemAdapter.get_json_schema(): https://github.com/scrapy/itemadapter?tab=readme-ov-file#class-method-get_json_schemaitem_class-type---dictstr-any
+
+* Added an ``"llmHint"`` JSON Schema metadata field to some item types and 
+  fields.
```

Comment:
Does it require https://github.com/zytedata/zyte-common-items/pull/136 to be merged?